### ISSUE TITLE
dapr init fails every other time on Windows

### ISSUE THREAD

**Issue by Haishi2016** [dapr/cli#219]
## Expected Behavior

dapr init should succeed when running on Windows

## Actual Behavior

dapr init fails every other time with a "file being used by another process" error

## Steps to Reproduce the Problem

Repeat "dapr init" multiple times on Windows

**Comment by artursouza** [comment]
Here is what I tried. It worked. I only got an error when I had an dapr app running in another console:
```
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr --version
CLI version: 0.3.0-rc.0
Runtime version: 0.3.0-rc.0

#After I start an app in dapr locally:


PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Error extracting Dapr binary: open c:\dapr/daprd.exe: The process cannot access the file because it is being used by another process.
```

**Comment by artursouza** [comment]
Tried again. Still, no error:

```
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
PS C:\Code\dapr_release\samples\2.hello-kubernetes\deploy> dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running
```

**Comment by Haishi2016** [comment]
This is what I saw on my machine:
-----------------------

D:\Go-2020\src\github.com>dapr --version
CLI version: 0.3.0-rc.0
Runtime version: edge

D:\Go-2020\src\github.com>dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Error moving Dapr binary to path: open c:\dapr/daprd.exe: The process cannot access the file because it is being used by another process.

D:\Go-2020\src\github.com>dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running

D:\Go-2020\src\github.com>dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Error moving Dapr binary to path: open c:\dapr/daprd.exe: The process cannot access the file because it is being used by another process.

D:\Go-2020\src\github.com>dapr init
Making the jump to hyperspace...
Downloading binaries and setting up components...
Installing Dapr to c:\dapr
Success! Dapr is up and running

-----

I just tried a few more times and I wasn't able to reproduce the issue anymore. I'll close this for now.
